gama/GamaClassifier.py

```diff
@@ -52,8 +146,8 @@ def _predict(self, x: pd.DataFrame):
         """
         y = self.model.predict(x)  # type: ignore
         # Decode the predicted labels - necessary only if ensemble is not used.
-        if y[0] not in list(self._label_encoder.classes_):
-            y = self._label_encoder.inverse_transform(y)
+        if y[0] not in list(self._label_encoder.classes_):  # type: ignore
+            y = self._label_encoder.inverse_transform(y)  # type: ignore
         return y
 
```

Comment:
@PGijsbers How do you pass mypy checks without adding these two type ignore here?

```
gama/GamaClassifier.py:149:29: error: "None" has no attribute "classes_"  [attr-defined]
gama/GamaClassifier.py:150:17: error: "None" has no attribute "inverse_transform"  [attr-defined]
```

Reply:
Ah, gotcha. I am not entirely sure. It's possible that these lines were never part of a PR since the introduction of `mypy` to the `pre-commit` pipeline. `pre-commit` only checks the changed files by default.

Reply:
This file did change, indeed, but not this function as you spotted. As a result, I believe I should either do an extra commit to solve this issue if you believe this is correct, but FYI this is outside of the PR's scope; however, on the other hand I could always not run the git hooks on this file but make sure that manually pre-commit checks do not say anything else that these issues expressed above 🤔

Your call, note that this will anyway have to wait until January because I will not work on this PR any further before the actual review 👌